libgssapi/src/context.rs

```diff
@@ -697,11 +699,35 @@ impl ClientCtx {
     /// number of times until step returns `Ok(None)`. At that point
     /// the context is fully initialized.
     pub fn step(&mut self, tok: Option<&[u8]>) -> Result<Option<Buf>, Error> {
```

Comment:
I think we should just cut 0.6. I'm more concerned with getting the right api in the long run than minimizing churn now. Seems like we should just add an optional 'channel_bindings' arg to step.

Reply:
:+1:  Patch incoming with the requested changes.